site/content/en/docs/Installation/Terraform/aks.md

```diff
@@ -41,7 +41,15 @@ kubectl get nodes
 ```
 
 Configurable parameters:
+
 - log_level - possible values: Fatal, Error, Warn, Info, Debug (default is "info")
+- cluster_name - the name of the AKS cluster (default is "agones-terraform-example")
+- agones_version - the version of agones to install (an empty string default is the latest version from the [Helm repository](https://agones.dev/chart/stable))
```

Comment:
Small nit:

```suggestion
- agones_version - the version of agones to install (an empty string, which is the default, is the latest version from the [Helm repository](https://agones.dev/chart/stable))
```

Same thing below as well. Otherwise, this looks good to merge :+1: